velodrome/token-counter/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM busybox:1.24.2
+ADD token-counter /
+ENTRYPOINT ["/token-counter"]
```

Comment:
Yeah, it doesn't really make a difference. But please don't make me do it the other way, I've already had that fight the other way around ...